models/org.go

```diff
@@ -432,7 +432,7 @@ func hasOrgVisible(e Engine, org *User, user *User) bool {
 		return true
 	}
 
-	if org.Visibility == structs.VisibleTypePrivate && !org.isUserPartOfOrg(e, user.ID) {
+	if org.IsOrganization() && (org.Visibility == structs.VisibleTypePrivate || user.IsRestricted) && !org.IsUserPartOfOrg(user.ID) {
```

Comment:
why change `isUserPartOfOrg` to `IsUserPartOfOrg` ?

Reply:
@mnsh :arrow_up:

Reply:
Good point - change reverted.